tools/sstableloader/src/org/apache/cassandra/tools/LoaderOptions.java

```diff
@@ -79,7 +78,6 @@ public class LoaderOptions
      * provide options instead of using these constant fields.
      * @deprecated Use {@code throttle-mib} instead
      */
-    /** @deprecated See CASSANDRA-17677 */
```

Comment:
@tolbertam are you sure about this? The fact we moved the class from dir to dir does not mean this stopped to be relevant.

Reply:
I think I dropped it because of the double javadoc.  I think I should just merge it into the doc above, e.g.:

```
@deprecated Use {@code throttle-mib} instead (see CASSANDRA-17677)
```

will make that change.